### ISSUE TITLE
Mustang: Settings updates

### ISSUE THREAD

**Issue by rjmackay** [ushahidi/platform#1063]
- Update each settings view to match new design
  - [x] roles
  - [x] categories
  - [x] users #1069
  - [x] general settings
  - [x] map settings
  - [x] data sources
  - [ ] import
- Implement new listing patterns for users, roles, categories - break into directives if needed
- Wait on filters refactoring tasks #1047 
- Site settings: Background becomes logo
- Out of scope:
  - Post type settings #1062

**Comment by rjmackay** [comment]
@middle8media do you think you could take this next and just update all the markup for the settings view, then flag any functionality thats not working?

**Comment by middle8media** [comment]
@rjmackay yes.

**Comment by willdoran** [comment]
These tasks need to be complete to support the settings conversion:
- [x] Master settings page
- [ ] Change image_header to deployment_logo
- [ ] Rebuild map settings management in settings-map-directive
- [ ] Change save button for datasources
- [ ] For general settings pattern, add cancel button to reset content on each page
- [ ] Mode context will be created per template
- [ ] Break custom select field into directive
- [ ] Break adaptive field into directive
- [ ] Break listing toolbar directive for controls
- [ ] Add cancel button functionality

**Comment by rjmackay** [comment]
Is this all complete now @willdoran @middle8media

**Comment by middle8media** [comment]
@rjmackay I am moving onto other tasks, but am available to help with settings as needed. My understanding is that @willdoran had a few more things to knock out, then we would be done.

**Comment by willdoran** [comment]
@rjmackay We're still waiting on the data configure screen which comes from post editor
